bin/windows/ze-ph-petstore.bat

```diff
@@ -0,0 +1,10 @@
+set executable=.\modules\swagger-codegen-cli\target\swagger-codegen-cli.jar
+
+If Not Exist %executable% (
+  mvn clean package
+)
+
+REM set JAVA_OPTS=%JAVA_OPTS% -Xmx1024M
+set ags=generate -i modules\swagger-codegen\src\test\resources\2_0\petstore.json -l ze-ph -o samples\client\petstore\ze-ph
```

Comment:
Sure, thanks for noticing) And sorry for not checking this "copy-paste" from bin/windows/php-petstore.bat thoroughly. For actual testing I used IDEA's "Run/Debug configuration".